Makefile

```diff
@@ -47,7 +47,7 @@ frontend: react-build
 
 .PHONY: setup
 setup:
-	pip install pip-tools pipenv black ;
+	pip install pip-tools pipenv "black == 21.6b0" "typing-extensions < 3.10" ;
```

Comment:
@AnOctopus I saw the discussion in Slack re: this issue. It seemed like you wanted to remove this step of installing black via pip here. Can you explain the context around why it's not sufficient to have black in the pipfile? Not blocking this PR, I just want to know so that I understand the setup better.

Reply:
I actually should double check if it needs to still be specified here. Initially it seemed to need it to help introduce constraints on the typing-extensions version but it wasn't sufficient alone and I haven't checked if it is needed still